package.json

```
@@ -41,7 +41,8 @@
     "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
     "dev:preview": "node playground/.output/server/index.mjs",
     "dev:preview:generate": "npx serve playground/dist",
-    "release": "npm run lint && npm run prepack && npx bumpp --push --tag --commit && npm publish --access=public && git push --follow-tags",
+    "prepublishOnly": "npm run lint && npm run prepack",
+    "release": "npx bumpp --push --tag --commit && npm publish --access=public",
```

```suggestion
    "release": "npx bumpp --push --tag --commit && npm publish",
```

It's only needed for the first time.